src/main/webapp/app/shared/fullscreen/fullscreen.component.ts

```diff
@@ -7,8 +7,6 @@ import { enterFullscreen, exitFullscreen, isFullScreen } from 'app/shared/util/f
     styleUrls: ['./fullscreen.scss'],
 })
 export class FullscreenComponent {
-    buttonIcon = 'compress';
```

Comment:
Why as IconProp in other files and writing it into template here?

Reply:
Because here 'compress' is a string but it needs to be of the Type IconProp. I guess explicitly casting it to a IconProp here would have also worked but I guess like this it's a bit more concise.